.goreleaser.yml

```diff
@@ -45,3 +45,16 @@ changelog:
 release:
   draft: true
   prerelease: auto
+brews:
+  -
+    name: tektoncd-cli
+    github:
+      owner: chmouel
+      name: homebrew-tektoncd-cli
+    folder: Formula
+    homepage: "https://github.com/tektoncd/cli"
+    description: A CLI for interacting with Tekton!
+    test: |
+      system "#{bin}/tkn --version"
+    install: |
```

Comment:
according to homebrew code, this should be implicit fyi : https://github.com/goreleaser/goreleaser/blob/3f082c9252eb93a4bdf5045e06accbde37bb6dc3/internal/pipe/brew/brew.go#L66-L80

Reply:
oh if I can remove it, I will !